packages/pluggableWidgets/datagrid-dropdown-filter-web/src/components/__tests__/DataGridDropdownFilter.spec.tsx

```diff
@@ -13,6 +13,10 @@ const commonProps = {
 };
 
 describe("Dropdown Filter", () => {
+    afterEach(() => {
+        delete (global as any)["com.mendix.widgets.web.UUID"];
```

Comment:
I understand that this is to reset the counter for every test, but it feels counterintuitive to the purpose of this PR. As the point is to allow multiple instances but with different IDs. Without this, will the counter still be the deterministic across multiple test runs? if so, i would advocate to removing this cleanup

Reply:
I keep the cleanup for tests with just one widget instance, so that tests with snapshots don't change when moving them around in the test file. I added a dedicated test to test uniqueness for all data filters.